include/wx/event.h

```diff
@@ -1946,31 +1955,92 @@ class WXDLLIMPEXP_CORE wxSetCursorEvent : public wxEvent
 
 private:
     wxDECLARE_DYNAMIC_CLASS_NO_ASSIGN(wxSetCursorEvent);
+};
+
+ // Touch Base Event
+
+class WXDLLIMPEXP_CORE wxTouchEventBase : public wxEvent
+{
+public:
+    wxTouchEventBase(wxWindowID winid = 0, wxEventType type = wxEVT_NULL)
+        : wxEvent(winid, type)
+    {
+    }
+
+    wxTouchEventBase(const wxTouchEventBase& event) : wxEvent(event)
+    {
+        m_pos = event.m_pos;
+    }
```

Comment:
Could/should be just
```suggestion
    wxTouchEventBase(const wxTouchEventBase& event) = default;
```